packages/trader/src/AppV2/Components/TradeParameters/Barrier/barrier.tsx

```diff
@@ -13,17 +13,14 @@ import { TTradeParametersProps } from '../trade-parameters';
 const Barrier = observer(({ is_minimized }: TTradeParametersProps) => {
     const {
         barrier_1,
-        onChange,
         duration_unit,
         is_market_closed,
-        setV2ParamsInitialValues,
         v2_params_initial_values,
```

Comment:
Do we need `v2_params_initial_values`? If we removed setter and logic of usage

Reply:
In case if you remove it, please, clean trade store as we have types for TV2ParamsInitialValues

Reply:
we are v2_params_initial_values in other trade-types as well so not sure if we can remove it from trade-store